cmd/pingo/main.go

```diff
@@ -50,18 +51,23 @@ func main() {
 		}
 		log.Info("received response within ", duration)
 	} else {
-
 		userStop := make(chan os.Signal)
-		loopStop := make(chan bool)
+		ctx := context.WithValue(context.Background(), "packageSlip", pingo.PackageSlip{
```

Comment:
While this is technically not wrong (stuffing your value in the context), but it introduces new issues to be aware of... for instance, when accessing that value from the context, you have to make sure it's there otherwise you risk a panic (see comment below).